tests/stdlib/tparsecfg.nim

```diff
@@ -1,23 +1,76 @@
 discard """
-  output: '''OK'''
+  targets: "c js"
 """
 
 #bug #6046
-import parsecfg
-
-var config = newConfig()
-config.setSectionKey("foo","bar","-1")
-config.setSectionKey("foo","foo","abc")
-config.writeConfig("test.ini")
-
-# test.ini now contains
-# [foo]
-# bar=-1
-# foo=abc
-
-var config2 = loadConfig("test.ini")
-let bar = config2.getSectionValue("foo","bar")
-let foo = config2.getSectionValue("foo","foo")
-assert(bar == "-1")
-assert(foo == "abc")
-echo "OK"
+import parsecfg, streams
+
+when not defined(js):
+  var config = newConfig()
+  config.setSectionKey("foo","bar","-1")
+  config.setSectionKey("foo","foo","abc")
+
+  config.writeConfig("test.ini")
+
+  # test.ini now contains
+  # [foo]
+  # bar=-1
+  # foo=abc
+
+  var config2 = loadConfig("test.ini")
+  let bar = config2.getSectionValue("foo","bar")
+  let foo = config2.getSectionValue("foo","foo")
+  assert(bar == "-1")
+  assert(foo == "abc")
+
+
+## Creating a configuration file.
+var dict1=newConfig()
+dict1.setSectionKey("","charset","utf-8")
```

Comment:
pre-existing, so optional to fix:

spacing after arguments `dict1.setSectionKey("", "charset", "utf-8")`
(+ elsewhere)